weasyl/controllers/content.py

```diff
@@ -30,58 +30,71 @@ def submit_(request):
 def submit_visual_get_(request):
     if not define.is_vouched_for(request.userid):
         raise WeasylError("vouchRequired")
-
-    form = request.web_input(title='', tags=[], description='', imageURL='', baseURL='')
-    if form.baseURL:
-        form.imageURL = urlparse.urljoin(form.baseURL, form.imageURL)
+    title = request.params.get('title', '')
+    description = request.params.get('description', '')
+    baseURL = request.params.get('baseURL', '')
+    imageURL = request.params.get('imageURL', '')
+    tags = request.params.get('tags', '').split(',')
```

Comment:
This is actually for Waxpost; the original call was correct.

https://github.com/Weasyl/waxpost-chrome/blob/cfe0ce252b60e4f0a2a3def94bec08f4c96abd5c/furaffinity.js#L23

```suggestion
    tags = request.params.getall('tags')
```